linetools/spectralline.py

```diff
@@ -407,7 +408,7 @@ def fill_data(self, trans, linelist=None, closest=False):
 
         # Update
         self.wrest = self.data['wrest']
-        self.trans = self.data['name']
+        self.name = self.data['name']
```

Comment:
I'm wondering now, should the name of the AbsLine contain redshift information? (Default could be z=0)